### PR TITLE
Soroush.Soroush+ version 1.0.30

### PR DESCRIPTION
**Pull request by Parsa2820**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/33145)

### CODE REVIEW

File: manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.locale.fa-IR.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: Soroush.Soroush+
+PackageVersion: 1.0.30
+PackageLocale: fa-IR
+Publisher: Soroush
+PublisherUrl: https://hi.splus.ir/
+PublisherSupportUrl: https://hi.splus.ir/contact
+PrivacyUrl: https://hi.splus.ir/privacy
+Author: Soroush
+PackageName: Soroush+
```
Comment (ItzLevvie) [inline manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.locale.fa-IR.yaml:12]:
```suggestion
PackageName: Soroush+ Desktop Application
```

File: manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.locale.fa-IR.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Soroush.Soroush+
+PackageVersion: 1.0.30
```
Comment (ItzLevvie) [inline manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.locale.fa-IR.yaml:5]:
```suggestion
PackageVersion: 1.0.30.0
```

File: manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Soroush.Soroush+
+PackageVersion: 1.0.30
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://windows.splus.ir/
+  InstallerSha256: 1A3EECA75C1BB3D836D67F17F22342692F8B90B7F5A48E7B78C69424B780FFA4
```
Comment (ItzLevvie) [inline manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.installer.yaml:10]:
```suggestion
  InstallerSha256: 1A3EECA75C1BB3D836D67F17F22342692F8B90B7F5A48E7B78C69424B780FFA4
  ProductCode: Soroush+_is1
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Soroush.Soroush+
+PackageVersion: 1.0.30
```
Comment (ItzLevvie) [inline manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.yaml:5]:
```suggestion
PackageVersion: 1.0.30.0
```

File: manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Soroush.Soroush+
+PackageVersion: 1.0.30
```
Comment (ItzLevvie) [inline manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.30.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.installer.yaml (hunk 5 above)
Comment (Parsa2820) [reply manifests/s/Soroush/Soroush+/1.0.30/Soroush.Soroush+.installer.yaml]: Suggestions were applied

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update the version folder as well.

**Comment by Parsa2820** [comment]
@ItzLevvie I can't rerequest review. Please review

**Review by ItzLevvie — APPROVED** [review]

**Comment by Parsa2820** [comment]
@jamespik  It seems reviewer with write access is also required in order to merge PR. Please review this

**Pull request merged**